src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -2320,13 +2320,8 @@ public void deleteTopic(Topic topic) {
   }
 
   // https://support.zendesk.com/hc/communities/public/posts/203464106-Managing-Organization-Memberships-via-the-Zendesk-API
-  public List<OrganizationMembership> getOrganizationMembershipByUser(long user_id) {
-    return complete(
-        submit(
-            req(
-                "GET",
-                tmpl("/users/{user_id}/organization_memberships.json").set("user_id", user_id)),
-            handleList(OrganizationMembership.class, "organization_memberships")));
+  public Iterable<OrganizationMembership> getOrganizationMembershipByUser(long user_id) {
```

Comment:
Nice catch, we could probably deprecate this method for later deletion, but outside of the scope of this PR.